SPECS/cdi/cdi.spec

```diff
@@ -16,9 +16,9 @@
 #
 
 Summary:        Container native virtualization
-Name:           containerized-data-importer
+Name:           cdi
```

Comment:
We have already published the package with the previous name, so we cannot now change it like that. Also, the way we usually do this kind of modification is by adding `Provides: cdi` to the spec instead. This way the scope of the changes is smaller as well (you only need to modify the spec file).

Reply:
You are correct! This is probably the right way to rename packages.  I have discarded previous commits and new commit with this "Provides" pushed.  Please review and advise.

Reply:
The new `Provides` looks good. Let's also do the same thing for the subpackages instead of re-naming them. Especially since now the new subpackages will be called `containerized-data-importer-apiserver`, so probably not what you're looking for anyway. ;)

Reply:
With old commits, which included renaming of containerized-data-importer.spec to cdi.spec, thrown out, the current single change does the necessary job - That single Provides creates all the packages packages with desired names :-) That is, the names of new packages created are cdi-apiserver, cdi-cloner, etc.